src/shared/constants.js

```diff
@@ -23,3 +23,20 @@ export const COLOR_WHITE = 'white';
 export const COLOR_DARK_GRAY = '#212121';
 export const COLOR_LIGHT_GRAY = '#eef2fb';
 export const COLOR_TRANSPARENT_LIGHT_GRAY = 'rgba(238, 242, 251, 0)';
+
+// languages
+export const supportedLanguages = [
```

Comment:
do you add languages manually here? Or is this also handled by transifex?

Reply:
We have to "activate" them manually in the app. This way transifex can send translations for a language that only has very limited content, and we can delay turning it on in the app. I don't think there's an easy way to automate this, unless we look at the files in the locales folder, and check how many lines they contain and all. I think it's not worth the effort, at least not now.